charts/seed-monitoring/charts/prometheus/templates/config.yaml

```diff
@@ -439,7 +439,67 @@ data:
 
     - job_name: vpn-connection
       honor_labels: false
+      metrics_path: /probe
       scrape_interval: 15s
+      params:
+        module:
+        - icmp_probe
+        target:
+        - 192.168.123.1
```

Comment:
Hard-coded value?

Reply:
I talked with @marwinski about this. 
@marwinski Could you please explain in detail why this would always work, when the vpn connection is healthy?

Reply:
Yes, this is a hardcoded value configured in https://github.com/gardener/vpn/blob/master/shoot/openvpn.config.template 

This is only for a private network between the two parties and not used for anything else. Well, it should probably be configurable in case somebody thinks of using that range for pods/nodes/services.